HoneyDo.Web/ClientApp/src/components/Tasks/Form.tsx

```diff
@@ -44,47 +64,63 @@ class TaskForm extends Component<TaskFormProps, TaskFormState> {
     }
   }
 
-  handleChange = () => (event: ChangeEvent<HTMLInputElement>) => {
+  handleNameChange = () => (event: ChangeEvent<HTMLInputElement>) => {
     const name = event.target.value;
     this.setState({ name, hasError: false });
   };
 
+  handleDateChange = (dates: Date[]) => {
+    const dueDate = dates.length > 0 ? dates[0] : undefined;
+    this.setState({ dueDate });
```

Comment:
You can combine this line and the line above:
```js
this.setState({ dueDate: dates.length > 0 && dates[0] ? dates[0] : null });
```
I would also go one step further and make sure it has data.

Reply:
Once again I don’t think that is easier to read. That’s what I try to got for.